includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -3735,6 +3736,16 @@ class AMP_Allowed_Tags_Generated {
 					'requires_extension' => array(
 						'amp-story',
 					),
+					'child_tags' => array(
```

Comment:
@todo: Needs updating `bin/amphtml.py` script to pull the descendants list etc automatically instead of editing the file directly.